src/components/pages/computing/NewInstancePage/cmp.tsx

```diff
@@ -64,6 +64,17 @@ export default function NewInstancePage({ mainRef }: PageProps) {
 
   // ------------------
 
+  const isBlockchainHoldingCompatible = useCallback(
```

Comment:
We should move it to SDK later (in evm_utils we have similar ones)

Reply:
I moved it to a domain file to be reusable, so we can handle the logic easier in the console page without needing to change to the sdk as it's not needed in another application by now.